zul/src/archive/web/js/zul/mesh/MeshWidget.js

```diff
@@ -1821,6 +1824,38 @@ zul.mesh.MeshWidget = zk.$extends(zul.Widget, {
 			if (pgib) hgh += pgib.offsetHeight;
 		}
 		return hgh ? hgh : defVal;
+	},
+	/**
+	 * Scroll to the specify item by giving index, use by Grid and Listbox,

```

Comment:
1. specified or specific
2. by the given index
3. used by Grid and Listbox.

Reply:
ok